text/0003-measure-metric-type.md

```diff
@@ -1,6 +1,6 @@
 # Consolidate pre-aggregated and raw metrics APIs
 
-**Status:** `proposed`
+**Status:** `accepted`
```

Comment:
accepted or approved? One of these is a typo.

Reply:
Fixed